Notification.Api/Services/NotificationService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Notification.Api.Models;
+using Quarentime.Common.Services;
+using System;
+using System.Threading.Tasks;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+
+namespace Notification.Api.Services
+{
+    public class NotificationService : INotificationService
+    {
+        private readonly IConfigurationService _configurationService;
+
+        public NotificationService(IConfigurationService configurationService)
+        {
+            _configurationService = configurationService;
+        }
+
+        public Task PushNotification()
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task SmsAlert(SmsAlertContract request)
```

Comment:
Rename this to be something more specific. 
We also need an endpoint for the invite notifications. 
We could make this SmsAlertContract more generic if we renamed the status to something like "Message" and then we can use only one endpoint for the other cases.